app/frontend/src/components/forms/manage/ApiKey.vue

```diff
@@ -45,6 +45,7 @@ if (canGenerateKey.value) {
 async function createKey() {
   loading.value = true;
   await formStore.generateApiKey(form.value.id);
+  filesApiAccess.value = apiKey.value?.filesApiAccess;
```

Comment:
The generateApiKey function in the form store will update the apiKey ref, so this just updates the value after the request.